File: java/Java/src/com/c/LinkList.java
package com.c;

public class LinkList {
	int data;
	LinkList next; 
	
	public LinkList(int data) {
		this.data=data;
	}
	
	public LinkList add(LinkList next) {
		LinkList rear = null;
		LinkList point;
		point = next;
		for(int i = 0 ;i<4;i++) {
			point.data=i;
			point=next.next;
		}
		point.next=rear;
		return next;
	}
	
	public void show(LinkList next) {
		LinkList tall =next.next;
		while (tall.next!=null) {
			System.out.println(tall.data);
			tall=tall.next;
		}
	}
	public static void main(String[] args) {
		LinkList linklist = new LinkList(1);
		linklist.add(next);
		linklist.show(next);
		
	}

}

File: C++/LinkList/LinkList.cpp
#include<iostream>
#include <stdlib.h>
struct Node
{
    int data;
    struct Node *next;
}LinkType;


class LinkList{
private:
    LinkType *head, *p, *q;
};


LinkType create(int n)
{
    LinkType *head, *p, *q;
    head = new Node[10];
    head->next=NULL;
    p=head;
    for(int i=0; i<n ; i++ ){
        q = new Node[10];
        cin>>q->data;
        q->next = NULL;
        p->next=q;
        p=q;
    }
    return head;
}

void display(LinkType *head){
    LinkType *tail;
    tail = head->next;
    while(tail!=NULL)
    {
        if(tail->next==NULL)
        cout<<"\n"<<tail->data;
        else
        cout<<" "<<tail->data;
        tail = tail->next;
    }

}

int main(){
    int n;
    cout<<"Input the length of linked list : ";
    cin>>n;
    LinkType *head;
    head = create(n);
    display(head);
   // printf("")
    //scanf("",)
    return 0;
}

